sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -913,12 +925,6 @@ public DigitalTwinsServiceVersion getServiceVersion() {
         return protocolLayer.getDigitalTwinModels().deleteWithResponseAsync(modelId, context);
     }
 
-    PagedFlux<String> listRelationships(String digitalTwinId, String relationshipName, Context context) {
```

Comment:
This function was sitting in the models API section, so I moved it back to the correct section